### PR TITLE
Tutorial.md improvements

### PR DESCRIPTION
**Pull request by ktzee**
**- What I did**
Followed the tutorial. Had some issues. Modified the doc adding the steps I've taken to progress.
"crictl create" was not working with the image path in container_redis.json so I changed it to the path that made it work for me.
**- How I did it**
Followed the tutorial, googled.

**- How to verify it**
Follow the tutorial on a fresh VM.

**- Description for the changelog**

Added clearer instructions to the tutorial.

### CODE REVIEW

File: test/testdata/container_redis.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		"name": "podsandbox1-redis"
 	},
 	"image": {
-		"image": "quay.io/crio/redis:alpine"
+		"image": "redis:alpine"
```
Comment (rhatdan) [inline test/testdata/container_redis.json]: Why did you change this?

File: tutorial.md [hunk 2]
```diff
@@ -307,13 +307,21 @@ First we need to setup a Pod sandbox using a Pod configuration, which can be fou
 cd $GOPATH/src/github.com/kubernetes-incubator/cri-o
 ```
 
+
+Copy the policy template into /etc/containers/:
+
+```
+sudo mkdir /etc/containers/
+sudo cp test/policy.json /etc/containers
```
Comment (rhatdan) [inline tutorial.md]: This is also available in skopeo-containers.

File: tutorial.md [hunk 3]
```diff
@@ -344,14 +352,13 @@ Annotations:
 
 ### Create a Redis container inside the Pod
 
-Use the `crictl` command to pull the redis image, create a redis container from a container configuration and attach it to the Pod created earlier:
+Use the `crictl` command to pull the redis image, create a redis container from a container configuration and attach it to the Pod created earlier while capturing the container ID:
 
 ```
 sudo crictl pull redis:alpine
```
Comment (rhatdan) [inline tutorial.md]: Maybe change this to quay.io/crio/redis:alpine

Comment (ktzee) [reply tutorial.md]: That would also make it so the change in container_redis.json is not necessary, I believe.

File: tutorial.md (hunk 2 above)
Comment (ktzee) [reply tutorial.md]: It probably gets copied over when installing from .deb, but when you build skopeo from source you find that missing. Does it make sense to leave that "check" in the tutorial? (Ubuntu 18.04 is out and the skopeo repo hasn't been updated yet, it seems).

Comment (rhatdan) [reply tutorial.md]: Maybe add a comment that if the file does not exist do the copy.

File: tutorial.md (hunk 3 above)
Comment (rhatdan) [reply tutorial.md]: Yes I hope so

Comment (ktzee) [reply tutorial.md]: Confirmed, it does. Reverting the change in container_redis.json and including this change here.

File: tutorial.md (hunk 2 above)
Comment (ktzee) [reply tutorial.md]: Makes sense. Thanks!

**Comment by rhatdan** [comment]
@ktzee You need to sign your commands and the header line can not be over 80 characters.

git commit -a --amend -s
git push --force

Should fix the issues.

**Comment by ktzee** [comment]
I apologize for the messy PR. It's my first PR ever.

File: test/testdata/container_redis.json (hunk 1 above)
Comment (TomSweeneyRedHat) [reply test/testdata/container_redis.json]: I'm guessing a merge issue....  But should be reverted.

**Comment by rhatdan** [comment]
@ktzee No problem, we all have to learn.

**Comment by rhatdan** [comment]
@ktzee Now you need to squash all of your commits together.  And remove the merge patch.

File: tutorial.md [hunk 4]
```diff
@@ -307,13 +307,21 @@ First we need to setup a Pod sandbox using a Pod configuration, which can be fou
 cd $GOPATH/src/github.com/kubernetes-incubator/cri-o
 ```
 
+
+In case the file /etc/containers/policy.json does not exist on your filesystem, copy it from the source tree:
+
+```
+sudo mkdir /etc/containers/
+sudo cp test/policy.json /etc/containers
```
Comment (TomSweeneyRedHat) [inline tutorial.md]: This is OK for a test system, but production wise I don't think you want to use this particular policy.json.  The better answer is to be sure that 'skopeo-constainers' was installed as noted at line 282.

**Comment by TomSweeneyRedHat** [comment]
@ktzee overall a nice PR, thanks for fixing this up and navigating through the git fun!

File: tutorial.md (hunk 4 above)
Comment (ktzee) [reply tutorial.md]: Yeah the policy seems to be a "bit insecure", but since we're executing all this on a throwaway VM for tutorial purposes it didn't look wrong. I can revert that back though. Once Ubuntu 18.04 .debs will be available for skopeo that will likely be useless anyway.

**Comment by ktzee** [comment]
I can't figure it out and I don't want to leave this mess here. Starting over :(
